### ISSUE TITLE
Seeking support for http broadcasts

### ISSUE THREAD

**Issue by in0finite** [saul/demofile-net#120]
I would like to add seeking support for http broadcasts. While connected, it would be possible to seek backwards (to beginning of broadcast ?), and then forwards up to latest received tick.

There are no `CDemoFullPacket`s in http broadcasts, so not sure how it would work.

I see there is `/full` endpoint, but it only delivers `CDemoPacket`. Is that good enough to restore full snapshot ?

`/full` endpoints could be enumerated by periodically calling `/sync`.

Do you maybe have some ideas how this could be implemented ?

**Comment by in0finite** [comment]
Ok, I just realized how it works : every fragment has it's own `/full` and `/delta` packets. 
So this is perfectly seekable. 
The only question is, can `DemoParser` be restored from `/full` fragment ?

**Comment by saul** [comment]
> The only question is, can DemoParser be restored from /full fragment ?

Yes it is - BUT from what I've seen the TTL on the /full and /delta packets is very short. IIRC it was less than a minute. They don't seem to stick around for long at all. With that in mind I'm not sure this is do-able.
